src/data/finset/gcd.lean

```diff
@@ -0,0 +1,129 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Aaron Anderson
+-/
+import data.finset.fold
+import data.multiset.gcd
+
+/-!
+# GCD and LCM operations on finsets
+
```

Comment:
Could you write a module doc following the [doc guidelines](https://leanprover-community.github.io/contribute/doc.html)?